core/templates/dev/head/components/search_results_directive.html

```diff
@@ -0,0 +1,25 @@
+<script type="text/ng-template" id="components/searchResults">
+  <div class="oppia-gallery-tiles-container">
+    <div ng-if="allExplorationsInOrder.length === 0" class="oppia-gallery-tiles-introduction">
+      <div class="oppia-gallery-no-match">
+        <img src="/images/general/no_explorations_found.png" alt="No explorations found">
+        <p style="text-align: center;">
+          Oops, it looks like your search did not match any explorations.<br><br>
+          <!-- NOTE: If you change the links here, please make the corresponding change for the 'Create New Exploration' button. -->
```

Comment:
Actually I think this comment can be removed now, since we've changed the overall routing of the site.